### DISCUSSION TITLE
A community-led mentorship initiative to help beginners in kickstarting their open-source journey by completing Hacktoberfest'21 challenge

### DISCUSSION THREAD

**Discussion by vinitshahdeo** [github/india#170] [Open Source]
### Hello community :wave:

I am Vinit Shahdeo, a [GitHub Star](https://stars.github.com/profiles/vinitshahdeo/) from :india:. I along with a few amazing [mentors](https://github.com/vinitshahdeo/Hacktoberfest2021/blob/main/MENTORS.md) have decided to contribute to mentor newbies in kickstarting their open-source journey and completing Hacktoberfest’21 challenge. We’re starting this open-source celebration journey with an optimistic vision of helping 1000 beginners.

## Project Maintainers

If you happen to be a project maintainer, we would be more than grateful to onboard you as a mentor. Please go through “[Muddle through Hacktoberfest 2021 like a Pro](https://vinitshahdeo.dev/hacktoberfest-2021)” blog post to learn more about the initiative.

- Add beginner-friendly issues to [`ISSUES.md`](https://github.com/vinitshahdeo/Hacktoberfest2021/blob/main/explore/ISSUES.md)

- Add your repositories to [`PROJECTS.md`](https://github.com/vinitshahdeo/Hacktoberfest2021/blob/main/explore/PROJECTS.md)

- Watch the mentor's [onboarding session](https://youtu.be/BMyVbZp8Gn8) on YouTube

- [Join us](https://github.com/vinitshahdeo/Hacktoberfest2021#mentoring) as a mentor

## Beginners

Here’s a [home](https://github.com/vinitshahdeo/Hacktoberfest2021) to your queries and quest for potential issues to contribute. We’ve compiled a handpicked list of [beginner-friendly issues](https://github.com/vinitshahdeo/Hacktoberfest2021/blob/main/explore/ISSUES.md) (& repositories) ahead of time so that you can get the ball rolling once Hacktoberfest kicks off! :rocket:

The [mentors](https://github.com/vinitshahdeo/Hacktoberfest2021/blob/main/MENTORS.md) will guide you on getting started with open-source and completing your Hacktoberfest challenge. 

- You can begin by introducing yourself in the [discussions](https://github.com/vinitshahdeo/Hacktoberfest2021/discussions/127). ✨ 

- Kindly [submit your details](https://forms.gle/GJSUTPBJ4Rw4DTMk9) if you're a beginner who's looking for a mentor. 🌈

## Support

This is a [community-led mentorship initiative](https://dev.to/vinitshahdeo/calling-out-beginners-for-a-community-led-mentorship-opportunity-for-hacktoberfest-16gh) to help beginners kickstarting their open-source journey completing Hacktoberfest'21 challenge. **Support us by spreading the word among newbies!**

[![Hacktoberfest 2021](https://github-readme-stats.vercel.app/api/pin/?username=vinitshahdeo&repo=Hacktoberfest2021)](https://github.com/vinitshahdeo/Hacktoberfest2021)

**Happy Hacktoberfest** :tada:

**Comment by FRTMentor** [comment]
Please reach out our support team.
support@futurereadytalent.in
